nekoyume/Assets/_Scripts/UI/Widget/ArenaBattlePreparation.cs

```diff
@@ -219,6 +204,52 @@ public override void Close(bool ignoreCloseAnimation = false)
 
         #endregion
 
+        private void UpdateAvatarState()
+        {
+            var avatarState = Game.Game.instance.States.CurrentAvatarState;
+            var arenaAvatarState = RxProps.PlayersArenaParticipant.Value.AvatarState;
+            var currentBlockIndex = Game.Game.instance.Agent.BlockIndex;
+
+            var arenaItems = arenaAvatarState.inventory.Items;
+            for (int i = arenaItems.Count - 1; i > 0; i--)
+            {
+                Nekoyume.Model.Item.Inventory.Item existItem;
+                switch (arenaItems[i].item)
+                {
+                    case Equipment arenaEquipment:
+                    {
+                        existItem = avatarState.inventory.Items.FirstOrDefault(item =>
```

Comment:
`avatarState.inventory.Items` can change to `avatarState.inventory.Equipments` ?

Reply:
To check `Locked` value later. type of `existItem` must be Item, not itemBase.